src/amoc_xmpp.erl

```diff
@@ -17,3 +33,48 @@ connect_or_exit(Spec) ->
             exit(connection_failed)
     end.
 
+%% @doc Picks a random server based on config var `xmpp_servers`
+%% This function expect a list of proplists defining the endpoint
+%% to which an XMPP client can connect, for instance:
+%% [[{host, "127.0.0.1"}, {port, 5222}], [{host, "127.0.0.1"}, {port, 5223}]]
+%% One of the above proplists is picked and can be added to user's spec.
+%% It's required that the proplists contains at least the `host` property.
+%% Since the `xmpp_servers` config option is read via `amoc_config` API,
+%% it's possible to pass it as an ENV var when starting amoc:
+%% > AMOC_xmpp_servers="[[{host,\"127.0.0.2\"}, {port, 5222}],[{host, \"127.0.0.1\"}, {port, 5223}]]" make console
+%% Default value, if xmpp_servers option is not set, is [{host, "127.0.0.1"}]
+-spec pick_server() -> [proplists:property()].
+pick_server() ->
+    Servers = amoc_config:get(xmpp_servers, [[{host, "127.0.0.1"}]]),
```

Comment:
I think the default should be passed in from the scenario, not assumed by the library, since it's more convenient for scenario readers.

In other words I imagine we ought to use:
- OS vars
- Erlang app vars
- a default from the scenario

instead of a hardcoded default. I.e. have `pick_server/1` instead of `pick_server/0`.

Reply:
Yes, it makes sense, I'll change `pick_server` to take the default value as an argument.
Currently the `amoc_config:get/2` works like you described, it first tries to read from the OS var, if it's not set it tries to read from the app vars and then returns default if the option was not in app vars.